scripts/cbmc_json_parser.py

```diff
@@ -20,6 +20,8 @@
 """
 
 import json
+import os
+
```

Comment:
Is there any reason for this separation of libraries? If we're grouping them, then maybe we should group `json, os, re,` and `sys` together as they're native libraries? 

Or is this newline a typo?

Reply:
Oh. Either my IDE or the autopep8 did that. Let me fix it.

Reply:
I figured out why there's a space. It's because the next import was using the `from <package> import <item>`. I split them to make cleaner.